kasa/smart/smartchilddevice.py

```diff
@@ -35,6 +35,14 @@ def __init__(
 
     async def update(self, update_children: bool = True):
         """Noop update. The parent updates our internals."""
+        # TODO: we should probably default update_children=False in the library
+        #  and perform these extra queries only on request.
+        if update_children:
```

Comment:
The above todo is something we should discuss about, if we change the update_children=False, we could modify `SmartDevice.update()` to call updates on children, too?

Reply:
Is this still a todo for 0.7.0?  I think that the default should probably be `False` and then we can always have the option on calling update on children directly when needed in the future.